ParabolInc/parabol
# chore(one-on-one): Add team exists warning

Fixes: #8562
Fixes: #8717
Fixes: #8814

- Adds a warning that an one-on-one team already exists within the org and will be reused (or new team will be created if not)
- Adds email validation
- Avatars added to the dropdown menu

Full demo: https://www.loom.com/share/cc57b4b73e3b4ff19ba4cf6c6a30805f

<img width="377" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/76104be7-0cf2-4ff4-a233-ea69d800532c">


<img width="380" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/7892aa7d-a41b-4ef5-aeac-a3eeb72f84b7">


**How to test:**

- Open one on one meeting page
- Enter custom email, see that a correct warning appears
- Select a user, see that a correct warning appears
- Select a user with whom you already had a meeting in one-on-one team, see that a correct warning appears (Note that now we also check one on one team by `isOneOnOneTeam` flag, so previously created teams might not be counted)
- See that a warning matches the an actual team that will be created/used
- Try to enter incorrect email, see it is not allowed
- See that you can still create simply check-in meeting without issues
